doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -420,7 +420,9 @@ To configure the T-shirt sizes (specified in [Table 4-17](../../../ref_model/cha
 | .2xlarge* | 8<br>-- vcpus 8 | 16 GB<br>-- ram 16384 | 160 GB<br>-- disk 160 |
 | .4xlarge* | 16<br>-- vcpus 16 | 32 GB<br>-- ram 32768 | 320 GB<br>-- disk 320 |
 
-In addition, to configure the storage IOPS the following two parameters need to be specified in the flavor create: --property quota:disk_write_iops_sec=<IOPS#> and --property quota:disk_read_iops_sec=<IOPS#>.
+In addition, to configure the storage IOPS the following two parameters need to be specified in the flavor create: --property quota:disk_write_iops_sec=<IOPS#> and --property quota:disk_read_iops_sec=<IOPS#>.  
+
+To configure SMT, the following parameter needs to be specified in the flavor create:  --property smt_enabled=True.
```

Comment:
Can you add a source of this statement? I am not able to find `smt_enabled` property in openstack flavor.

Reply:
@deebhatia This was provided by me. My mistake -- needed to be clearer. As you know, in the flavor extra-specs it is possible to assign extra key-value properties. This statement needs to be accompanied by further details (reason why it was not added to the table). The main intent is to be able to have host aggregates with hosts with smt enabled and smt disabled (latter being necessary for secure workloads).

Reply:
@deebhatia on further thought, given that, both flavors require SMT to be enabled (no choice for SMT to be disabled), I will suggest to Scott to remove the statement.

Reply:
It is not an issue in keeping it. it is just that for custom flags like host aggregate flags, we should either explicitly mention or prefix them `aggregate_instance_extra_specs` or do both. Currently, it looks like a standard flavor extra specs which might confuse the reader.

Reply:
@deebhatia deleted offending sentence.